src/app/videochat/video-chat.component.ts

```diff
@@ -38,13 +39,25 @@ interface IHighlight {
   current: Feed;
 }
 
+console.log([
+    ThumbnailsModeButtonComponent,
+    MainFeedComponent,
+    PushToTalkComponent,
+    FeedComponent,
+    ScreenShareButtonComponent,
+    LeaveButtonComponent,
+    NotificationComponent,
+    ChatComponent
+  ]);
+
+
 @Component({
   selector: "jh-video-chat",
   template: require("./video-chat.component.html"),
   directives: [
     ThumbnailsModeButtonComponent,
     MainFeedComponent,
-    PushToTalkComponent,
+    //PushToTalkComponent,
```

Comment:
When commenting a component, it would be nice to put a FIXME or something. FIXME and TODO are really useful to grep for temporary stuff.